Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -232,11 +234,20 @@ private void OnDisable()
             StopCoroutine(mPhysicsCoroutine);
         }
 
-        void OnSceneLoaded(Scene scene, LoadSceneMode mode) { if (mFrameStack.Count > 0) ManualUpdate(); }
-        void OnSceneUnloaded(Scene scene) { if (mFrameStack.Count > 0) ManualUpdate(); }
-
+        void OnSceneLoaded(Scene scene, LoadSceneMode mode) 
+        { 
+            if (Time.frameCount == m_LastFrameUpdated && mFrameStack.Count > 0)
+                ManualUpdate();
+        }
+        void OnSceneUnloaded(Scene scene)
+        {
+            if (Time.frameCount == m_LastFrameUpdated && mFrameStack.Count > 0)
+                ManualUpdate();
+        }
+        
```

Comment:
Yes, because it means that the async callback happened after ManualUpdate was called for the frame, so the cameras could be stale.  If the callback comes in before ManualUpdate has been called this frame, then we want to avoid calling it twice, which was the cause of the jitter